app/models/image.rb

```diff
@@ -84,6 +84,18 @@ class Image < ActiveFedora::Base
     index.as :stored_searchable, :facetable
   end
 
+  property :nul_creator, predicate: ::Vocab::Donut.has_creator, multiple: true do |index|
```

Comment:
should this be `::Vocab::Donut.creator`?

(same for subject and contributor)

Reply:
I was going with has, although using underscores, to follow this general concept:

https://samvera.github.io/customize-metadata-model.html#to-add-a-new-multi-value-property

If we have a DONUt way though I'm happy to default to that

Reply:
Doesn't it need to match what you added to the Donut vocab?